testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/AbstractClientTest.java

```diff
@@ -87,6 +89,7 @@ public static ClientRepresentation createClientRep(String clientId, String proto
         attributes.put(SAML_SIGNATURE_ALGORITHM, "RSA_SHA256");
         attributes.put(SAML_FORCE_NAME_ID_FORMAT, "false");
         attributes.put(SAML_NAME_ID_FORMAT, "username");
+        attributes.put(SamlConfigAttributes.SAML_ARTIFACT_BINDING_IDENTIFIER, ArtifactBindingUtils.computeArtifactBindingIdentifierString("saml"));
```

Comment:
Is this line really necessary? `SAML_ARTIFACT_BINDING_IDENTIFIER` should be automatically set on server when not present

Reply:
If I understand the test correctly, these attributes are just checked after the creation, they are not set when creating the client.

Reply:
It seems they are used also for creation, see e.g. `org.keycloak.testsuite.console.clients.ClientSettingsTest.createSAML()`
This line can stay here if there is another test (e.g. in base testsuite) that tests the creation of SAML client without this attribute and then checks that the attribute has been added by keycloak server when reading the client back.

Reply:
From `newClient` `ClientRepresentation` only `clientId` and `protocol` is used for creation, see [this](https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/other/console/src/main/java/org/keycloak/testsuite/console/page/clients/CreateClientForm.java#L27-L31).